src/MissionManager/PlanManager.cc

```diff
@@ -394,8 +394,8 @@ void PlanManager::_handleMissionItem(const mavlink_message_t& message, bool miss
         param2 =        missionItem.param2;
         param3 =        missionItem.param3;
         param4 =        missionItem.param4;
-        param5 =        (double)missionItem.x / qPow(10.0, 7.0);
-        param6 =        (double)missionItem.y / qPow(10.0, 7.0);
```

Comment:
@DonLakeFlyer looks like you used two time missionItem.x

Reply:
Ah crap, thanks

Reply:
I need to switch the unit tests to MISSION_ITEM_INT so they will catch this

Reply:
@FlavioTonelli Thanks so much for catching this

Reply:
Just lucky that I scrolled into this pull request ;D